src/app/modules/startup/controller.nim

```diff
@@ -649,3 +655,30 @@ proc inputConnectionStringForBootstrapping*(self: Controller, connectionString:
 
 proc setLoggedInAccount*(self: Controller, account: AccountDto) =
   self.accountsService.setLoggedInAccount(account)
+
+proc setLoggedInProfile*(self: Controller, publicKey: string) =
+  debug "<<< setLoggedInProfile", publicKey
+  self.loggedInPofilePublicKey = publicKey
+
+proc getLoggedInAccountPublicKey*(self: Controller): string =
+  return self.loggedInPofilePublicKey
+
+proc getLoggedInAccountDisplayName*(self: Controller): string =
+  return self.accountsService.getLoggedInAccount().name
+
+proc getLoggedInAccountImage*(self: Controller): string =
+  let iamges = self.accountsService.getLoggedInAccount().images
+  for img in iamges:
+    if img.imgType == "large":
+      return img.uri
+  return ""
+
+proc biometricsSupported*(self: Controller): bool =
+  return main_constants.IS_MACOS
```

Comment:
:sob:

Reply:
We already have defined const for this, maybe we could just pick a better name for it. Check here https://github.com/status-im/status-desktop/blob/master/src/constants.nim#L7